appveyor.yml

```diff
@@ -13,7 +13,7 @@ environment:
       # For this package-template, we include examples of Cython modules,
       # so Cython is required for testing. If your package does not include
       # Cython code, you can set CONDA_DEPENDENCIES=''
-      CONDA_DEPENDENCIES: "Cython"
+      CONDA_DEPENDENCIES: "Cython shapely"
```

Comment:
The Appveyor build fails because shapely is not available in this conda channel:
https://ci.appveyor.com/project/Astropy/regions/build/1.0.74/job/wfb95tev3c94p9jh#L81

@astrofrog - Do you know how to fix this? If no, please remove shapely here and let's move the Windows shapely install issue to a separate future issue.